src/game/overlay.c

```diff
@@ -400,12 +395,20 @@ void Overlay_RemoveAmmoText(void)
 
 void Overlay_DrawAmmoInfo(void)
 {
-    const double scale = 0.8;
+    const double scale = 1.5;
     const int32_t text_height = 17 * scale;
     const int32_t text_offset_x = 3;
-    const int32_t screen_margin_h = 20;
+    const int32_t screen_margin_h = 24;
     const int32_t screen_margin_v = 18;
 
+    // Scale ammo string properly if text scaling used.
+    double scale_ammo_to_bar = g_Config.ui.bar_scale;
+    if (g_Config.ui.text_scale > 1) {
+        scale_ammo_to_bar /= g_Config.ui.text_scale;
+    } else if (g_Config.ui.text_scale < 1) {
+        scale_ammo_to_bar *= 1 / g_Config.ui.text_scale;
+    }
```

Comment:
Wait, how are these two operations different? (Is `1` getting promoted to a double `1.0`?)

Reply:
Oops you are right these are the same. I was testing some different formulas lol. I'll do:
```
// Scale ammo string properly if text scaling used.
double scale_ammo_to_bar =g_Config.ui.bar_scale / g_Config.ui.text_scale;
```